source/clear-linux/tutorials/dlaas-benchmarks.rst

```diff
@@ -0,0 +1,186 @@
+If you need to install the containers-basic bundle, use :command:`swupd` to do so.
+
+.. code-block:: bash
+
+  sudo swupd bundle-add containers-basic
+
+
+To run Kubernetes on |CL|, please refer to the :ref:`kubernetes` tutorial to ensure it is correctly installed and configured.
+
+We have validated these steps against the following software package versions
+
+  * |CL| 26240
+  * Docker 18.06.1
+  * Kubernetes 1.11.3
+  * Go 1.11.12
+
+
+TensorFlow Benchmarks
+=====================
+
+This section describes running the `TensorFlow benchmarks`_ as run by the TensorFlow community. These steps provide a template to run other benchmarks, providing they can invoke TensorFlow.
+
+The |CL| Deep Learning Stack is available in two versions.  First, a version that includes TensorFlow* optimized for Intel Architecture, and second, a version that includes the TensorFlow* framework optimized using Intel® Math Kernel Library for Deep Neural Networks (Intel® MKL-DNN) primitives.
+
+Download and run either the `TensorFlow`_ or the `TensorFlow with MKL-DNN`_  docker image from hub.docker.com. The next commands will take place in the running container. Replace <docker_name> with the name of the image.
+
+First, clone the benchmark repository
+
+.. code-block:: bash
+
+  docker exec -t <docker_name> bash -c ‘git clone http://github.com/tensorflow/benchmarks -b cnn_tf_v1.11_compatible’
+
+Next, execute the benchmark script to run the benchmark
+
+.. code-block:: bash
+
+  docker exec -i <docker_name> bash -c ‘python benchmarks/scripts/tf_cnn_benchmarks/tf_cnn_benchmarks.py --device=cpu --model=resnet50 --data_format=NWHC ’.
+
+You can replace the model with one of your choice supported by the TensorFlow benchmarks.
+
+
+
+Kubeflow Benchmarks
+===================
+
+The benchmark workload will run in a Kubernetes (k8s) container. We will use Kubeflow and deploy three nodes for this tutorial, to get a decent return.
+
+k8s setup
++++++++++
+
+Please follow the instructions in the :ref:`kubernetes` tutorial to get set up on |CL|.  The k8s community also has `instructions`_ if you are using Ubuntu instead of |CL|.
+
+
+k8s networking
+++++++++++++++
+
+We have used `flannel`_ as the network provider for these tests. If you are comfortable with another network layer, there is nothing that will prevent you from using that.
+Refer to the Kubernetes `networking documentation`_ for setup.
+
+
+Images
+++++++
+
+We will need to create a docker image that will include launcher.py to include the |CL| Deep Learning Stack, and put the benchmarks repo in the right location. From the docker image, run the following
+
+.. code-block:: bash
+
+  mkdir -p /opt
+  git clone https://github.com/tensorflow/benchmarks.git /opt/tf-benchmarks
+  cp launcher.py /opt
+  chmod u+x /opt/*
+
+Your entry point then becomes "/opt/launcher.py".
+
+
+This will build an image which can be consumed directly by TFJob from kubeflow.  We are working to create these images as part of our release cycle.
```

Comment:
This comment is misleading. We mentioned a set of instructions that must be ran on the image, but never tell the user how to actually build and run it on a container using Docker. Also, we must mention our jsonnet files should point to this image. That being said, I think this section should have the following structure:
1. How to build: steps for building a docker image given a Dockerfile
2. Mention the image should be pushed to a hub (i.e. dockerhub) for the TFJobs to consume
3. Modify the .jsonnet files to point to that image

Reply:
@bd-dean, we should consider adding more detail her per @DnPlas 's suggestion.